src/main/java/seedu/address/model/person/Person.java

```diff
@@ -259,6 +259,14 @@ public PersonBuilder withInteractions(Set<Interaction> interactions) {
             return this;
         }
 
+        /**
+         * Adds the {@code Interaction} to the {@code Set<Interaction>} of the {@code Person} that we are building.
+         */
+        public PersonBuilder addInteraction(Interaction interaction) {
+            this.interactions.add(interaction);
+            return this;
+        }
+
```

Comment:
Should we keep it to `withInteractions` do `addInteraction` instead?

Reply:
I think it's good to have both for flexibility.